opentelemetry-sdk/src/opentelemetry/sdk/metrics/_internal/export/__init__.py

```diff
@@ -519,7 +530,13 @@ def _shutdown():
             return
 
         self._shutdown_event.set()
-        self._daemon_thread.join(timeout=(deadline_ns - time_ns()) / 10**9)
+        if (
+            self._export_interval_millis > 0
+            and self._export_interval_millis < math.inf
+        ):
```

Comment:
nit to avoid duplicating the condition above

```suggestion
        if this._daemon_thread:
```